src/deluge/gui/menu_item/lfo/sync.h

```diff
@@ -20,21 +20,28 @@
 #include "model/song/song.h"
 #include "processing/sound/sound.h"
 
-namespace deluge::gui::menu_item::lfo::global {
+namespace deluge::gui::menu_item::lfo {
 
 class Sync final : public SyncLevel {
 public:
-	using SyncLevel::SyncLevel;
+	Sync(uint8_t lfoId, deluge::l10n::String name, deluge::l10n::String type) : SyncLevel(name, type), lfoID(lfoId) {}
```

Comment:
Other note on the constructors: we usually keep the `l10n::String` parameters first.

Reply:
Gotcha. I figured putting the shared ID in the beginning was easier to read:
```
lfo::Type lfo1TypeMenu{LFO1_ID, STRING_FOR_SHAPE, STRING_FOR_LFO1_TYPE};
lfo::Rate lfo1RateMenu{LFO1_ID, STRING_FOR_RATE, STRING_FOR_LFO1_RATE, params::GLOBAL_LFO_FREQ};
lfo::Sync lfo1SyncMenu{LFO1_ID, STRING_FOR_SYNC, STRING_FOR_LFO1_SYNC};
```
vs
```
lfo::Type lfo1TypeMenu{STRING_FOR_SHAPE, STRING_FOR_LFO1_TYPE,, LFO1_ID};
lfo::Rate lfo1RateMenu{STRING_FOR_RATE, STRING_FOR_LFO1_RATE, params::GLOBAL_LFO_FREQ, LFO1_ID};
lfo::Sync lfo1SyncMenu{STRING_FOR_SYNC, STRING_FOR_LFO1_SYNC, LFO1_ID};
```
Should I change to the latter?

Reply:
I think so, yeah. That's also easier to handle when I eventually convert all this stuff to the new menu compiler logic...